MdePkg/Test/MdePkgHostTest.dsc

```diff
@@ -57,3 +57,4 @@
   MdePkg/Test/Mock/Library/GoogleTest/MockMemoryAllocationLib/MockMemoryAllocationLib.inf
   MdePkg/Test/Mock/Library/GoogleTest/MockPciExpressLib/MockPciExpressLib.inf
   MdePkg/Test/Mock/Library/GoogleTest/MockUefiDevicePathLib/MockUefiDevicePathLib.inf
+  MdePkg/Test/Mock/Library/GoogleTest/MockPciLib/MockPciLib.inf

```

Comment:
nit: looks like there's extra whitespace before the text

Reply:
I checked the format, and it looks the same as the previous line.